sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -10,6 +10,13 @@
 
 ### Other Changes
 
+- Removed the code of internal vendored instrumentation libraries `OpenTelemetry.Instrumentation.AspNetCore` and `OpenTelemetry.Instrumentation.Http`.
```

Comment:
As we are adding these 2 points, we may need to explain the impacts. If there are no impact, let us call that out.